tsdb/querier.go

```diff
@@ -819,7 +819,7 @@ func (s *populatedChunkSeries) Next() bool {
 			c.Chunk, s.err = s.chunks.Chunk(c.Ref)
 			if s.err != nil {
 				// This means that the chunk has be garbage collected. Remove it from the list.
-				if s.err == ErrNotFound {
+				if s.err == storage.ErrNotFound {
```

Comment:
I think all of this should be `errors.Cause`, so we are safe on potential wraps someday.

Reply:
:+1: